src/components/ProductCard/styles.module.scss

```diff
@@ -0,0 +1,124 @@
+.productCard {
+  width: 308px;
+  background-color: #FFFFFF;
+  border: 1px solid #F1F3F3;
+  border-radius: 6px;
+}
+
+.imgContainer {
+  position: relative;
+  width: 100%;
+  height: 210px;
+  background: #FFFFFF;
+}
+
+.productImg {
+  object-fit: fill;
+  width: 306px;
+}
+
+.discountBadge {
+  width: 73px;
+  height: 59px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 8px 12px;
+  background: #FDCA40;
+  border-radius: 6px;
+  position: absolute;
+  top: 100%;
+  right: 5%;
+  transform: translateY(-50%);
+}
+
+.discountMessage {
+  font-family: 'DM Sans';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  text-align: center;
+}
+
+.discountPercentage {
+  font-family: 'DM Sans';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.productInfo {
+  padding: 0 5%;
+}
+
+.pricesSection {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  width: 130px;
+  height: 70px;
+}
+
+.listPrice {
+  font-family: 'DM Sans';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 22px;
+  text-decoration-line: line-through;
+  color: #A0ACAF;
+  align-self: center;
+  margin-right: 5px;
+}
+
+.discountedPrice {
+  font-family: 'DM Sans';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 26px;
+  line-height: 34px;
+  margin-left: 5px;
+}
+
+.productDescription {
+  font-family: 'DM Sans';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  margin: 0;
+}
+
+.productColors {
+  font-family: 'DM Sans';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 24px;
+  color: #A0ACAF;
+  margin-top: 0;
+}
+
+/* .offerButton {
+  width: 166px;
+  height: 41px;
+  padding: 10px 45px;
+  background: #FF4F17;
+  border-radius: 4px;
+  position: absolute;
+  top: 100%;
+  right: 5%;
+  transform: translateY(-75%); 
+
+  font-family: 'DM Sans';
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 21px;
+  text-align: center;
+  color: #FFFFFF;
+} */
```

Comment:
Commented code

Reply:
This code is commented out because I will use it when I implement the extended ProductCards.